### ISSUE TITLE
Previous Versions

### ISSUE THREAD

**Issue by 1destruction1** [si1nnx/psxx#1]
Can you re-release the previous version with the Alien Forest GUI, this version from 17 hrs ago seem to be older with only the steampunk chest as the latest auto farm available, thank you.

**Comment by Therogamer** [comment]
Yes, it seems as if everything remained only the farming locations got removed
